### PR TITLE
Attempt to add schemes tutorial

### PR DESCRIPTION
**Pull request by HereAround**
As requested by @HechtiDerLachs , this is an attempt to bring in the first schemes tutorial. As such,
- Had to create a new colored QR code.
- Create a couple of new files.

I should believe that this PR works, but could not verify yet. In particular, @aaruni96 if I run jekyll locally, then it gives me an empty list of tutorials. Maybe I forgot to run a local script?

@HechtiDerLachs I did not have much time for this PR, as you know. So I just chose black for the color of the schemes QR Code. Do you like that? If not, can you please create a new colored QR code and update this PR? (Or send me the new QR code that you like better.)

@HechtiDerLachs @aaruni96 I could not open this notebook yesterday without logging into github. Can you try if this notebook can be reached without a password? If a password (or OSCAR authorization of any sort) is needed, then please do **not** merge this PR -- I think the tutorials should be accessible even for people without github account/OSCAR rights.

### CODE REVIEW

File: _data/examples.yml [hunk 1]
```diff
@@ -38,6 +38,18 @@
 
 
 
+# Tutorials hosted at https://github.com/oscar-system/notebooks
+
+- title: Elliptic Fibrations and Covered Schemes
+  repository: oscar-system/notebooks/
+  branch: master
+  filename: notebooks/elliptic_fibrations/elliptic_fibrations_and_covered_schemes.ipynb
```
Comment (aaruni96) [inline _data/examples.yml]:
```suggestion
  filename: notebooks/elliptic_fibrations/elliptic_fibrations_and_covered_schemes
```

**Comment by aaruni96** [comment]
> Maybe I forgot to run a local script

You need to run the script which fetches tutorial's last update times before any tutorials show up at all

> Can you try if this notebook can be reached without a password

It cannot. The oscar-system/notebooks is a private repository, so, you need to be specifically granted access to this, even just logging into github is not enough.

**Comment by HereAround** [comment]
> > Can you try if this notebook can be reached without a password
> 
> It cannot. The oscar-system/notebooks is a private repository, so, you need to be specifically granted access to this, even just logging into github is not enough.

Thank you for verifying. This I believe is a strict no-go. The tutorials should be accessible to everybody. So ping to @simonbrandhorst and @HechtiDerLachs . If you really want this as a tutorial, you either need to move it to a different location or change the access rights to the file.

Let me mention that we test the tutorials once every 24 hours with the **latest stable** OSCAR release. Does this notebook run with this version of OSCAR?

We can also not necessarily fix failures/errors in those notebooks (is it just a coding issue? a mathematical issue? the code runs successfully but presents a mathematically incorrect result). Rather we need your help. If you change the file location/access writes, consider having convenient write access so that you can provide fixes with the least effort in the future.

Let us know how you want to proceed.

**Comment by HereAround** [comment]
One more thing to consider:

Each tutorial topic in OSCAR comes with a colored QR-Code. By now, already a fair amount of colors are taken (cf. https://www.oscar-system.org/tutorials/). For now, I have just created a black QR Code. If you care for a different color, can you please generate the QR code in your favorite color. The content of the QR code should simply be the link https://www.oscar-system.org/tutorials/Schemes/, where the scheme(s) tutorial(s) will appear in the future.

File: _data/examples.yml [hunk 2]
```diff
@@ -38,6 +38,18 @@
...
 
+# Tutorials hosted at https://github.com/oscar-system/notebooks
+
+- title: Elliptic Fibrations and Covered Schemes
+  repository: oscar-system/notebooks/
+  branch: master
+  filename: notebooks/elliptic_fibrations/elliptic_fibrations_and_covered_schemes
+  author: Simon Brandhorst, Matthias Zach
+  thumbnail: Schemes.png
+  language: julia
```
Comment (HechtiDerLachs) [inline _data/examples.yml]: Hello everyone! 

I am very sorry, but I realize only now that I must have sent the wrong link to @HereAround ! 

It should really be [this one here](https://github.com/HechtiDerLachs/OscarTutorials/blob/main/covered_schemes_and_elliptic_fibrations/elliptic_fibrations_and_covered_schemes.ipynb). That has also been accessible without password during the ICMS. 

My apologies again for all the effort you have probably all put into this already.

File: _data/examples.yml [hunk 3]
```diff
@@ -38,6 +38,18 @@
 
 
 
+# Tutorials hosted at https://github.com/oscar-system/notebooks
+
+- title: Elliptic Fibrations and Covered Schemes
+  repository: oscar-system/notebooks/
```
Comment (aaruni96) [inline _data/examples.yml]:
```suggestion
  repository: HechtiDerLachs/OscarTutorials
```

File: _data/examples.yml [hunk 4]
```diff
@@ -38,6 +38,18 @@
...
+  branch: master
+  filename: notebooks/elliptic_fibrations/elliptic_fibrations_and_covered_schemes
```
Comment (aaruni96) [inline _data/examples.yml]:
```suggestion
  filename: covered_schemes_and_elliptic_fibrations/elliptic_fibrations_and_covered_schemes
```

**Review by aaruni96 — CHANGES_REQUESTED** [review]
This should then be workable

**Comment by HechtiDerLachs** [comment]
> If you care for a different color, can you please generate the QR code in your favorite color.

I don't know how to do that, unfortunately. But I would probably go for the color with html code `db58bf`.

**Comment by aaruni96** [comment]
Here it is, you'll need to add it to the PR yourself though @HereAround (I don't yet know how to push things to someone else's PR)

schemes-colord

**Comment by HereAround** [comment]
Ok. Just updated everything. (I did not bother to keep the history straight. Just squashing this one, once we are done.)

Let me verify that this PR is now functional...

**Comment by HereAround** [comment]
Ok. This seems to work locally for me. (I cannot open the notebook in nbviewer, but this might be a problem with nbviewer and/or the internet connection in KL.)

@aaruni96 Any obvious mistakes? If not, I suggest we squash and merge this.

**Comment by aaruni96** [comment]
Looks good to me

**Comment by HereAround** [comment]
Just merged. @HechtiDerLachs , please verify that it looks and works as you desire. Let me know if changes are needed.

**Comment by HechtiDerLachs** [comment]
Thx! 

The `nbviewer` also doesn't work for me (404). But I can access the github version.

**Comment by aaruni96** [comment]
Ah, the branch is wrong (its `master` in the website, but `main` on github). Github web transparently redirects you to the default branch, nbviewer probably can't

@HereAround just requires `master` to be changed to `main`, and then it should work

**Comment by HereAround** [comment]
> Ah, the branch is wrong (its `master` in the website, but `main` on github). Github web transparently redirects you to the default branch, nbviewer probably can't
> 
> @HereAround just requires `master` to be changed to `main`, and then it should work

Ok. Let me adjust this.

**Comment by HereAround** [comment]
https://github.com/oscar-system/oscar-website/pull/367

**Pull request merged**